src/ReactiveUI.Tests/ObservableAsPropertyHelperTest.cs

```diff
@@ -152,6 +152,23 @@ public void OAPHDeferSubscriptionParameterDefersSubscriptionToSource()
         }
 
         [Fact]
+        public void OAPHDeferSubscriptionParameterIsSubscribeIsNotTrueInitially()
```

Comment:
minor typo here (missing 'd'), but I had to read this several times for it to click